<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowAGRenderer</class>
 <widget class="QMainWindow" name="MainWindowAGRenderer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Renderizador Genetico</string>
  </property>
  <property name="windowIcon">
   <iconset resource="cebolla.qrc">
    <normaloff>:/img/imagenes/icono.png</normaloff>:/img/imagenes/icono.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_3"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>21</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <widget class="QMenu" name="menu_Imagen">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>&amp;Imagen</string>
     </property>
     <addaction name="action_Cargar_Imagen"/>
     <addaction name="action_Guardar_Imagen"/>
    </widget>
    <widget class="QMenu" name="menu_ADN">
     <property name="title">
      <string>&amp;ADN</string>
     </property>
     <addaction name="action_Guardar_ADN"/>
    </widget>
    <addaction name="menu_Imagen"/>
    <addaction name="menu_ADN"/>
    <addaction name="action_Configurar"/>
    <addaction name="separator"/>
    <addaction name="action_Salir"/>
   </widget>
   <widget class="QMenu" name="menu_Ayuda">
    <property name="title">
     <string>A&amp;yuda</string>
    </property>
    <addaction name="action_Acerca_de_Renderizador_Genetico"/>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menu_Ayuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Configurar">
   <property name="text">
    <string>&amp;Configurar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Salir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Cargar_Imagen">
   <property name="text">
    <string>&amp;Cargar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Guardar_Imagen">
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Cargar_2">
   <property name="text">
    <string>&amp;Cargar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Guardar_ADN">
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Acerca_de_Renderizador_Genetico">
   <property name="text">
    <string>&amp;Acerca de Renderizador Genetico</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="cebolla.qrc"/>
 </resources>
 <connections/>
</ui>
